core/src/lib.rs

```diff
@@ -24,18 +24,15 @@
 // #[macro_use]
 extern crate alloc;
 
-// Use `wee_alloc` as the global allocator.
-#[global_allocator]
-static ALLOC: wee_alloc::WeeAlloc = wee_alloc::WeeAlloc::INIT;
-
 #[cfg(all(test, feature = "std"))]
 #[macro_use]
 mod test_utils;
 
-#[cfg(not(feature = "std"))]
-mod panic_handler;
-
```

Comment:
So I guess this functionality is now handled by `node-runtime`. Is there a way to control it from pDSL?